webapp/components/test-runs.js

```diff
@@ -49,6 +53,11 @@ const TestRunsQueryLoader = (superClass, opt_queryCompute) =>
       return /(\.(html|htm|py|svg|xhtml|xht|xml)(\?.*)?$)/.test(path);
     }
 
+    computePathIsASubfolder(path) {
+      return !this.computePathIsATestFile(path)
+        && path && path.split('/').filter(p => p).length > 0;
```

Comment:
If paths are normalized at some point and a case like "/////" can't happen, then this could simply check `path !== '/'`, right? Doesn't matter much, just a thought.